src/CellData/DomainContributions.jl

```diff
@@ -1,8 +1,10 @@
 
 """
 """
+using DataStructures
```

Comment:
You need to move this up, above the docstring (which is currently empty).  Just make the `using` statement the first line of the file.